src/main/java/org/powerbot/script/rt4/GrandExchangeItem.java

```diff
@@ -0,0 +1,94 @@
+package org.powerbot.script.rt4;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.StandardCharsets;
+
+public class GrandExchangeItem {
+	private int id;
+	private String name;
+	private int dailyVolume;
+	private int guidePrice;
+	private boolean members;
+	private int buyLimit;
+
+	public final static GrandExchangeItem NIL = new GrandExchangeItem();
+
+	public GrandExchangeItem(String name) {
+		this.name = name;
+		try {
+			String itemNameURL = name.replaceAll(" ", "_");
+			URLConnection connection = new URL("https://oldschool.runescape.wiki/w/Exchange:" + itemNameURL).openConnection();
```

Comment:
There's actually an API here - https://oldschool.runescape.wiki/w/RuneScape:Real-time_Prices

Reply:
added GrandExchangeItem#initLivePrices in addition